src/ddmd/cppmangle.d

```diff
@@ -603,139 +709,74 @@ public:
          * e        long double, __float80
          * g        __float128
          * z        ellipsis
+         * Dd       64 bit IEEE 754r decimal floating point
+         * De       128 bit IEEE 754r decimal floating point
+         * Df       32 bit IEEE 754r decimal floating point
+         * Dh       16 bit IEEE 754r half-precision floating point
+         * Di       char32_t
+         * Ds       char16_t
```

Comment:
Do we need to document types we don't support?

Reply:
I'd prefer that wchar and dchar map to them, but that would break backward compatibility. Not sure what to do here, so I documented it.